docs-website/sidebars.js

```diff
@@ -242,6 +242,11 @@ module.exports = {
           type: "doc",
           id: "docs/domains",
         },
+        {
+          label: "File Upload and Download in Documentation",
```

Comment:
This should be called "Data Documentation" instead. 

Have a feature page for one sub-feature of documentation does not sound right to me